### PR TITLE
Rollup + buble + buba

### PR DESCRIPTION
**Pull request by TylorS**
Updates build system to use rollup and buble
Update tests to use buba/register
Update coverage to use istanbul 1.1.0 alpha for code coverage with buba
Add `jsnext:main` field for next-gen bundlers to take advantage of ES6 builds

### CODE REVIEW

**Comment by davidchase** [comment]
LGTM

**Comment by TylorS** [comment]
@briancavalier Does this look alright to you? :)

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import buble from 'rollup-plugin-buble'
+
+export default {
+  entry: 'src/index.js',
+  dest: 'dist/prelude.js',
+  format: 'umd',
+  moduleName: 'most',
```
Comment (davidchase) [inline rollup.config.js]: i think the moduleName here should be `mostPrelude`

Comment (TylorS) [inline rollup.config.js]: :+1:

**Comment by davidchase** [comment]
LGTM

File: package.json [hunk 2]
```diff
@@ -29,20 +31,18 @@
...
     "eslint-config-standard": "^5.1.0",
     "eslint-plugin-promise": "^1.1.0",
     "eslint-plugin-standard": "^1.3.2",
-    "isparta": "^4.0.0",
+    "istanbul": "^1.1.0-alpha.1",
     "jsinspect": "^0.7.2",
     "mocha": "^2.4.5",
     "power-assert": "^1.4.1",
-    "rollup": "^0.25.3",
+    "rollup": "^0.25.8",
```
Comment (briancavalier) [inline package.json]: Can we use the newest rollup, or do we need to stick with an older version?  (0.34.7 as of right now)

**Comment by TylorS** [comment]
@mostjs/maintainers I updated all the dependencies :)

**Comment by davidchase** [comment]
still LGTM :)

**Comment by briancavalier** [comment]
lgtm

**Comment by briancavalier** [comment]
Hmmm, is lgtm broken?

**Comment by TylorS** [comment]
LGTM

**Comment by TylorS** [comment]
Might be

**Comment by briancavalier** [comment]
lgtm seems to be down.  Merging.

**Pull request merged**